packages/design-tokens/design-token-previews/components/formatTokenName.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
We already have something like this on the `TokenName` component. Are we still using this function?